firefox_puppeteer/api/prefs.py

```diff
@@ -202,3 +202,13 @@ def set_pref(self, pref_name, value):
             """, script_args=[pref_name, value])
 
         assert retval
+
+    def persist_prefs(self):
+        """Persist current pref values to the default prefs file (typically prefs.js)
+        """
+        with self.marionette.using_context('chrome'):
+            self.marionette.execute_script("""
+              Cu = Components.utils;
+              Cu.import("resource://gre/modules/Services.jsm");
+              Services.prefs.savePrefFile(null);
```

Comment:
I still don't understand this. [savePrefFile](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsIPrefService#savePrefFile%28%29) will store the changes in prefs.js if the parameter as given is null. But that should be done anyhow at latest when Firefox closes. Why are changed preferences not been written out to the prefs.js file when we call `set_pref()`?